composer.json

```diff
@@ -18,15 +18,15 @@
         "publishing"
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^7.4 || ^8.0",
         "silverstripe/framework": "^4.6",
         "silverstripe/cms": "^4",
         "silverstripe/config": "^1",
         "symbiote/silverstripe-queuedjobs": "^4.5",
         "silverstripe/versioned": "^1.6"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7",
```

Comment:
I recommend using a testing recipe here instead of explicit modules.
```
"silverstripe/recipe-testing": "^1 || ^2",
```

with that you can remove both `phpunit/phpunit` and `squizlabs/php_codesniffer`